src/engine/controls/cuecontrol.cpp

```diff
@@ -135,6 +135,10 @@ CueControl::CueControl(const QString& group,
     m_pCueIndicator = new ControlIndicator(ConfigKey(group, "cue_indicator"));
     m_pPlayIndicator = new ControlIndicator(ConfigKey(group, "play_indicator"));
 
+    m_pIsPlaying = new ControlPushButton(ConfigKey(group, "is_playing"));
```

Comment:
This can be made read only by using connectValueChangeRequest() and forceSet()
This can also be a plain ControlObject.